src/components/Home/Home.js

```diff
@@ -92,8 +96,7 @@ const Home = () => {
   const handleDateChange = (date) => {
     setSelectedDate(date);
     setVaccineData([]);
```

Comment:
Why though? @Justinnn07

Reply:
Thats required, because when we are changing the date, we need to clear the list right? thats a feature!

Reply:
@Justinnn07 `setVaccineData([])` is required. It stores the fetched data.

Reply:
@Yash-271120 yep yep thats the one, please dont remove it!

One final thing please remove your package lock so that we can finally merge your branch